### DISCUSSION TITLE
What is Github? I just want to know how to get Wallet Connect 2

### DISCUSSION THREAD

**Discussion by Journey557** [community/community#59404] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don’t have a clue what Github is, I was directed here from Discord. I want to know how to get the new version of Wallet Connect. I just read about the change on June 25th. I’m a senior with 2 wallets a TRUST and ZENGO wallets and I bought 4 presales. I tried to collect one today but my wallet wouldn’t connect. I desperately need help, please? I don’t understand the lingo here or anything that I’ve read all day. I asked for help on discord and was scammed, my TRUST wallet has been emptied. He said he was connecting me to Wallet Connect 2. I don’t even understand what that is or the reason for the change. I had such a difficult time just getting wallets and learning how to buy a presale. I don’t know anything else. I don’t know why my wallet wouldn’t connect today the (yesterday now) the 26th- I wasted the entire day trying to connect and trying to figure out how to get the Wallet Connect2. Please help. Many thanks in advance. Journey 557
I don’t know what a repository is. I’m so lost and running out of time less than 24 hours to get this new wallet connect and no understanding of anything I’ve read on this site at all. I’m just looking for someone honest to guide me 😭please?

**Comment by Journey557** [comment]
I came here for technical support that is what Discord posted. I can’t find any Support or CHAT . How does a person get help. . I just want to know how to get Wallet Connect 2 before it is too late and I lose all my presales. I don’t understand why there is a change, why we won’t be able to use our wallets. If I z knew this a month ago I would never have got involved with crypto at all. I’ve lost so much money all ready.😢 is there any SUPPORT here? I don’t understand why I was sent here for help if there is NO SUPPORT TEAM. I need assistance please, I’m totally lost. Journey 557

**Comment by queenofcorgis** [comment]
Hi @Journey557 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Wallet Connect. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Journey557** [reply]
I'm not going back to Discord for help. I asked yesterday now over 38 hours straight trying to update to v2 or v3. When I asked for help an imposter said he was Moderator/Admin He had me sign into my Trust wallet said "he would upgrade for me so I could connect with the new wallet connect to the  4 presale purchases I've made. Well he helped all right- helped himself to my Trust Wallet and emptied it! Why don't you have Support with a Legit Customer Support Team? All the Scammers are on Discord and Twitter,as soon as you ask for help they know you are new and begin to prey on you and steal.😭All I want to know is how to get my Zengo & Trust Wallets updated so that they will work June 28- connect to my pre-sales.I don't know what I'm doing- don't even know what a D'APP is I'm 66 years old and I'm self taught but this is way over my head. What happens to all 4 pre-sales, you just end up with my coins and money? I'm overwhelmed, devastated and in tears 😭. Can't get anyone honest to help & I'm totally lost!PLEASE I'm begging you to help me. Many thanks in advance.
Gail McGhee
[redacted due to PII]

**Reply by Journey557** [reply]
Wallet Connect has sent me here as I'm trying desperately to update my wallet connect to v2 or v3- only hours remaining. Why was I sent here if you won't help me 😢. I'm 66 years old and SO VERY LOST! I need someone to advise how to get the update and my Zengo wallet is currently Frozen so I can't change wallet connect. I just can't win- absolutely no idea why wallet connect sent me here. I don't even understand the terminology. All I did was buy 4 presale meme coins on my iphone 12. Praying that one will hit big and I can retire my 69 yr old Cancer Survivor (twice) husband from the painful 13 hour days he must work F.T. since he had back to back Cancer for 5 years and now I don't know if I can collect my coins.

**Reply by queenofcorgis** [reply]
@Journey557 unfortunately, you were given the wrong information - GitHub and the community's focus is on the GitHub features and platform and coding, we are not able to assist at all in your case.  Wallet Connect does host a project on GitHub so checking them out here: https://github.com/WalletConnect may be helpful. 

The best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
